spec/services/flow/steps/proceedings_sca/heard_as_alternatives_step_spec.rb

```diff
@@ -1,17 +1,45 @@
 require "rails_helper"
 
 RSpec.describe Flow::Steps::ProceedingsSCA::HeardAsAlternativesStep, type: :request do
-  let(:legal_aid_application) { create(:legal_aid_application) }
+  let(:legal_aid_application) { build_stubbed(:legal_aid_application) }
 
   describe "#path" do
-    subject(:path) { described_class.path.call(legal_aid_application) }
+    subject(:path) { described_class.path.call(legal_aid_application, parameters) }
 
-    it { is_expected.to eql providers_legal_aid_application_heard_as_alternatives_path(legal_aid_application) }
+    context "when the proceeding is passed direct" do
+      let(:parameters) { { proceeding: } }
+      let(:proceeding) { build_stubbed(:proceeding, ccms_code: "PB020") }
+
+      it { is_expected.to eql providers_legal_aid_application_heard_as_alternative_path(legal_aid_application, proceeding) }
+    end
+
+    context "when the proceeding is passed as part of an options hash" do
+      let(:parameters) { { heard_together: false, proceeding: } }
+      let(:proceeding) { build_stubbed(:proceeding, ccms_code: "PB020") }
+
+      it { is_expected.to eql providers_legal_aid_application_heard_as_alternative_path(legal_aid_application, proceeding) }
+    end
   end
 
   describe "#forward" do
-    subject(:path) { described_class.forward }
+    subject(:path) { described_class.forward.call(legal_aid_application, proceeding) }
+
+    context "and the proceeding is a Prohibited steps order" do
+      let(:proceeding) { build_stubbed(:proceeding, ccms_code: "PB020") }
```

Comment:
Is PB020 not Specific Issue Order (I think PB019 is Prohibited Steps Order)

Reply:
Great catch, I have updated the stubbed proceeding :+1: